### PR TITLE
doc: Add missing indexes/blockfilter/basic to doc/files.md

### PR DESCRIPTION
**Pull request by maflcko**

### CODE REVIEW

File: doc/files.md [hunk 1]
```diff
@@ -46,6 +46,7 @@ Subdirectory       | File(s)               | Description
 `blocks/`          | `revNNNNN.dat`<sup>[\[2\]](#note2)</sup> | Block undo data (custom format)
 `chainstate/`      | LevelDB database      | Blockchain state (a compact representation of all currently unspent transaction outputs and some metadata about the transactions they are from)
 `indexes/txindex/` | LevelDB database      | Transaction index; *optional*, used if `-txindex=1`
+`indexes/blockfilter/basic/` | LevelDB database      | Blockfilter index for the basic filtertype; *optional*, used if `-blockfilterindex=basic`
```
Comment (hebasto) [inline doc/files.md]: `indexes/blockfilter/basic/` contains:
- `db` subdir which holds a LevelDB database
- `fltrNNNNN.dat` filter files

**Review by hebasto — COMMENTED** [review]
Concept ACK.

File: doc/files.md (hunk 1 above)
Comment (MarcoFalke) [reply doc/files.md]: Right, good catch

**Comment by hebasto** [comment]
ACK fa191c058f00116bb844e59e42e24ad961e19cbe

nit: the substring "LevelDB database" could be dropped from the "Description" column ;)

**Review by fanquake — APPROVED** [review]
ACK fa191c058f00116bb844e59e42e24ad961e19cbe

**Comment by MarcoFalke** [comment]
> nit: the substring "LevelDB database" could be dropped from the "Description" column ;)

Done

**Pull request merged**